certified-connectors/RamQuest Actions - UAT/apiProperties.json

```diff
@@ -1,7 +1,34 @@
 {
     "properties": {
-        "connectionParameters": {},
-        "iconBrandColor": "#55346D",
+        "connectionParameters": {
+            "token": {
+                "type": "oAuthSetting",
+                "oAuthSettings": {
+                    "identityProvider": "oauth2",
+                    "scopes": [
+                        "good-service offline_access"
+                    ],
+                    "redirectMode": "GlobalPerConnector",
+                    "customParameters": {
+                        "AuthorizationUrl": {
+                            "value": "https://sm.uat.ortitletech.com/auth/realms/supermarket/protocol/openid-connect/auth"
+                        },
+                        "TokenUrl": {
+                            "value": "https://sm.uat.ortitletech.com/auth/realms/supermarket/protocol/openid-connect/token"
+                        },
+                        "RefreshUrl": {
+                            "value": "https://sm.uat.ortitletech.com/auth/realms/supermarket/protocol/openid-connect/token"
+                        },
+                        "RedirectUrl": {
+                            "value": "https://global.consent.azure-apim.net/redirect/ramquestactionsuat"
+                        }
+                    },
+                    "clientId": "pacman"
+                },
+                "uiDefinition": null
+            }
+        },
+        "iconBrandColor": "#007ee5",

```

Comment:
please make sure you are using the iconBrandColor for your connector. Icon color must contain a non-transparent, non-white color (#ffffff) background and not-default color (#007ee5) that matches your specified icon background color. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector).

Reply:
@vmanoharas sorry, copy paste mistake, Icon brand color updated now Pls approve & Merge